src/main/kotlin/org/koitharu/kotatsu/parsers/site/madara/Madara5Parser.kt

```diff
@@ -176,9 +176,9 @@ abstract class Madara5Parser @InternalParsersApi constructor(
 		source = source,
 	)
 
-	@MangaSourceParser("MANGAOWLS", "BeautyManga", "en")
-	class MangaOwls(context: MangaLoaderContext) : Madara5Parser(context, MangaSource.MANGAOWLS, "beautymanga.com") {
+	@MangaSourceParser("BEAUTYMANGA", "BeautyManga", "en")
```

Comment:
Enum value name should not be changed due to compatibility

Reply:
Then pr closed?

Reply:
A class name may be updated. I forgot about it)